### PR TITLE
 Refactor HP print pdf

### LINKED ISSUE #5098: Hearing Worksheet | Fix formatting on the Print/Save as PDF version
As a judge, I would like to be able to see as much as possible of the Hearing Worksheet on one page. 
When printing/save as PDF view, the note boxes are printing on the page 2 and a lot of white space is on Page 1. This happens in both demo and UAT. We heard this feedback from users as well.

Recent updates to Caseflow Commons have wreaked a bit of havoc on the Print view so this ticket will also address those.

## AC
1. Remove any automatic page breaks... e.g. The Hearing Summary section should be right below the Issues section, not pushed to Page 2.
screen shot 2018-04-06 at 3 34 40 pm
screen shot 2018-04-06 at 3 34 16 pm

**Update/fix the .pdf formatting to reflect the following:**
2. [Veteran FirstName LastName’s] Hearing Worksheet = h1
screen shot 2018-06-07 at 4 49 25 pm
3. Labels in all caps should be ~ 1/3 smaller than the item it is labeling
screen shot 2018-06-07 at 4 48 08 pm
4. Veteran/Appellant Information and Hearing Summary = h2 (see previous screenshot)
5. Veteran/Appellant Information and Hearing Summary = h2 (see previous screenshot).
screen shot 2018-06-07 at 5 23 49 pm
6. Shrink the type on 'Representative Name', 'Witness...', and 'Periods and...' to be the same size as 'Procedural History' and 'BVA Decision'.
screen shot 2018-06-07 at 5 24 13 pm
7. More space is required between some rows above Procedural History in order to ensure a clear relationships. e.g in the screenshot below more space is needed between the 'Ewald D. Buckridge' (appellant name) and the label 'Representative Name' because they are not related.
screen shot 2018-06-07 at 4 56 40 pm
screen shot 2018-06-07 at 5 01 21 pm
8. Remove the box around and icon next to the Veteran ID.
screen shot 2018-06-07 at 5 19 19 pm
9. Remove empty space between elements.
screen shot 2018-06-07 at 4 59 19 pm
10. In the Procedural History section, tighten the spacing between Appeal Stream 1 and the first BVA decision date so that it is the same as that between 'BVA Decision' and NOD.  
screen shot 2018-06-07 at 5 09 03 pm
11. Hide the Procedural History 'box' and expand/collapse icon.
screen shot 2018-06-07 at 5 09 03 pm

_**Current**_
screen shot 2018-06-06 at 10 40 12 pm

_**Desired**_
screen shot 2018-06-07 at 5 25 54 pm

### PR DESCRIPTION
**Pull request by Sjones352**
Resolves #5098

### Description
This PR shows the numbering of the print pdf. 

### Acceptance Criteria 
- [ ] The Notes section should be right below the Issues section, not pushed to Page 2.
- [ ] Update/fix the .pdf formatting to reflect the following: [Veteran FirstName LastName’s] Hearing Worksheet = h1
- [ ] Labels in all caps should be ~ 1/3 smaller than the item it is labeling
- [ ] Veteran/Appellant Information and Hearing Summary = h2
- [ ] Veteran/Appellant Information and Hearing Summary = h2
- [ ]  Shrink the type on 'Representative Name', 'Witness...', and 'Periods and...' to be the same size as 'Procedural History' and 'BVA Decision'
- [ ] More space is required between some rows above Procedural History in order to ensure a clear relationships. e.g in the screenshot below more space is needed between the 'Ewald D. Buckridge' (appellant name) and the label 'Representative Name' because they are not related. 
- [ ] Remove the box around and icon next to the Veteran ID.
- [ ]  Remove empty space between elements.

### Testing Plan
1. Go to print pdf and check all the above acceptance criteria. 
screen shot 2018-06-12 at 2 32 58 pm

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -15,11 +15,11 @@
  *= require _commons.scss
  *= require _tabs.scss
  *= require reader/_reader_styles_root.scss
- *= require hearings/_hearings_styles_root.scss
  *= require _pdf_viewer.css
  *= require _style_guide.scss
  *= require react/_react_styles_root.scss
  *= require dispatch/_dispatch.scss
  *= require _hearings.scss
+ *= require hearings/_hearings_styles_root.scss
```
Comment (sharonwarner) [inline app/assets/stylesheets/application.css]: Can we recheck out this file? It looks like it's just a line number change.

File: client/app/hearings/HearingWorksheet.jsx [hunk 2]
```diff
@@ -145,9 +143,9 @@ export class HearingWorksheet extends React.PureComponent {
         />
       </form>
       {this.props.print &&
-        <WorksheetFooter
-          veteranName={this.props.worksheet.veteran_fi_last_formatted}
-        />
+          <WorksheetFooter
+            veteranName={this.props.worksheet.veteran_fi_last_formatted}
+          />
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: Same with these few lines, we should avoid updating lines unnecessarily.

File: client/app/hearings/HearingWorksheet.jsx [hunk 3]
```diff
@@ -3,7 +3,6 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import HearingWorksheetStream from './components/HearingWorksheetStream';
-import PrintPageBreak from '../components/PrintPageBreak';
```
Comment (sharonwarner) [inline client/app/hearings/HearingWorksheet.jsx]: Do we use this component anywhere else? We can probably delete the entire component.

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Sjones352) [reply app/assets/stylesheets/application.css]: Done

File: client/app/hearings/HearingWorksheet.jsx (hunk 2 above)
Comment (Sjones352) [reply client/app/hearings/HearingWorksheet.jsx]: Done

**Pull request merged**